_sass/charts.scss

```diff
@@ -3,11 +3,11 @@ rect.bar {
   mix-blend-mode: difference;
 }
 
-text {
-  fill: rgb(71, 71, 71);;
-  mix-blend-mode: luminosity;
-  font-family: 'Source Code Pro', monospace;
-}
+// text {
```

Comment:
curious about this change -- was this to allow the multiple colours for the graph legends?